<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_16">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>290</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_14">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="date_time_date_time_lb">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>334</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="date_time_info">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>405</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Display Info</string>
   </property>
  </widget>
  <widget class="Line" name="line_17">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>428</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_time_time_le">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>508</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>오전 07:52:10</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_time_date_label">
   <property name="geometry">
    <rect>
     <x>654</x>
     <y>352</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Date : </string>
   </property>
  </widget>
  <widget class="QLabel" name="date_time_time_label">
   <property name="geometry">
    <rect>
     <x>654</x>
     <y>380</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_time_date_time_le">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>448</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>2019-10-11 오후 11:25:48</string>
   </property>
  </widget>
  <widget class="QPushButton" name="date_time_set_time">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>568</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="date_time_edit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>308</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="date_time_date_lb">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>352</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="date_time_set_date_time">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>538</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>DateTime</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_time_date_le">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>478</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>2019-12-07</string>
   </property>
  </widget>
  <widget class="QPushButton" name="date_time_set_date">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>538</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_time_date_time_label">
   <property name="geometry">
    <rect>
     <x>654</x>
     <y>334</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DateTime : </string>
   </property>
  </widget>
  <widget class="QLabel" name="date_time_time_lb">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>378</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
